invenio_records_lom/utils/metadata.py

```diff
@@ -270,7 +299,33 @@ def set_datetime(self, datetime: str) -> None:
 
     ###############
     #
-    # methods for manipulating LOM's `technical` category
+    # methods for manipulating LOM's `metametadata` (3) category
+    #
+    ###############
+
+    def append_metametadata_contribute(
+        self,
+        name: str,
+        url: str,
+        logo: str,
+        role: str,
+    ) -> None:
+        """Append metametadata contribute."""
+        contribute = {
+            "role": {
```

Comment:
the form
```py
{
  "source": langstringify(..., lang="x-none"),
  "value": langstringify(..., lang="x-none"),
}
```
is common in LOM.
so common in fact, that the LOM-standard gave it it's own name: `vocabulary`
because it's so common, we also got a direct function for this `vocabularify(role)` (takes source as optional argument, which defaults to "LOMv1.0")

could use `vocabularify` here
if you prefer the current version for its explicitness, leaving it is fine too...